doc/source/tutorial/stats/resampling.rst

```diff
@@ -0,0 +1,96 @@
+.. _resampling-and-monte-carlo-introduction:
+
+Resampling and Monte Carlo Methods
+==================================
+
+.. currentmodule:: scipy.stats.resampling
+
+Introduction
+------------
+
+Resampling and Monte Carlo methods are statistical techniques that
+replace mathematical analysis with lots of computation.
+
+For example, suppose you and your brother Kyle find yourselves
+hitchhiking down a long and lonesome road. All of a sudden, there shines
+a shiny demon… in the middle… of the road. And he says:
```

Comment:
These ellipses strike me as unintentional (and maybe undesired?  Can't come up with a reason why aside from letting the markdown engine do the pretty-formatting itself)

Reply:
Listen to the song : )

Reply:
I meant that ellipsis is represented as a single character rather than three separate periods

Reply:
Oh yeah, another case of automatic conversion from notebook to restructured text.

Reply:
```suggestion
a shiny demon... in the middle... of the road. And he says:
```